README.md

```diff
@@ -154,12 +175,11 @@ In the following section we guide you trough installing and running a Gazebo sim
 
    ```bash
    cd ~
-   git clone https://github.com/PX4/Firmware.git
+   git clone https://github.com/PX4/Firmware.git --recursive
    cd ~/Firmware
-   git submodule update --init --recursive
    ```
 
-1. Install PX4 dependencies. A complete list is available on the [PX4 Dev Guide](http://dev.px4.io/en/setup/dev_env_linux_ubuntu.html#common-dependencies).
+1. Install PX4 dependencies. A complete list is available on the [PX4 Dev Guide](http://dev.px4.io/en/setup/dev_env_linux_ubuntu.html#common-dependencies). 
```

Comment:
We no longer list the common dependencies but instead use a script (at this URL) called (https://github.com/PX4/Firmware/blob/master/Tools/setup/ubuntu.sh). To set up my environment I just removed `gazebo9` and `libgazebo9-dev` from [ubuntu.sh](https://github.com/PX4/Firmware/blob/master/Tools/setup/ubuntu.sh#L185). 

What is best setup in your opinion? Will all those dependencies in the gazebo block be installed by other setup instructions? ie if I have gazebo as a separate sim installation should I just remove the ones I did, or everything?

Reply:
yes, I think this script is good minus the gazebo install. If you have it installed by the script and then install the full ros melodic version some parts get screwed up and gazebo doesn't work.

Reply:
I added more explicit instructions